.build/external-documentation/cli-getting-started.quickstart.yml

```diff
@@ -4,6 +4,14 @@ metadata:
   name: docs-cli-getting-started
   annotations:
     order: 1
+  labels:
+    service-category/application-services: true
+    product/rhosak: true
+    product/rhosr: true    
+    product/rhoad: true
```

Comment:
We don't have any CLI support for RHOAD yet, so you should probably remove (or at least comment out) this line for now.

Reply:
+1

Reply:
removed to avoid confusion

Reply:
Still need to remove the rhoad entry?